src/lib/holocene/checkbox.svelte

```diff
@@ -86,51 +99,34 @@
       class:indeterminate
       {...omit($$restProps, 'data-testid')}
     />
-    <span class="checkmark" class:hoverable class:on-dark={onDark}>
-      {#if indeterminate}
-        <Icon class="absolute top-0 left-0 h-4 w-4" name="hyphen" />
-      {:else if checked}
-        <Icon
-          class="absolute top-0 left-0 h-4 w-4"
-          name="checkmark"
-          strokeWidth={3}
-        />
-      {/if}
-    </span>
   </label>
 </div>
 
 <style lang="postcss">
   .checkbox {
-    @apply block h-[18px] w-[18px] cursor-pointer select-none text-sm leading-[18px] text-primary;
+    @apply flex cursor-pointer select-none items-center gap-3 text-sm leading-[18px] text-primary;
```

Comment:
On the designs, it looks like this should be `items-start` ?
<img width="702" alt="Screenshot 2024-01-09 at 2 14 09 PM" src="https://github.com/temporalio/ui/assets/11775628/217976b9-6e7a-4449-9362-a54b55f0147c">